test/gtest/uct/tcp/test_tcp.cc

```diff
@@ -286,5 +286,20 @@ UCS_TEST_P(test_uct_tcp, check_addr_len)
     EXPECT_EQ(sizeof(uct_tcp_ep_addr_t), iface_attr.ep_addr_len);
 }
 
+UCS_TEST_P(test_uct_tcp, get_pci_bw)
+{
+    double expected_pci_bw;
+    uct_iface_attr_t iface_attr;
+
+    expected_pci_bw = uct_tcp_iface_get_pci_bw(GetParam()->dev_name.c_str());
+    if (expected_pci_bw == DBL_MAX) {
+        UCS_TEST_SKIP_R("Device is not an IB device");
+    }
+    ucs_status_t status = uct_iface_query(m_ent->iface(), &iface_attr);
+    ASSERT_UCS_OK(status);
+
+
+    ASSERT_EQ(iface_attr.bandwidth.shared, expected_pci_bw);
```

Comment:
the test seems wrong. if we had bug in reading sysfs the test would be skipped (because get_pci_bw returns DBL_MAX) and not fail.
we should probably test this:
- for any iface, (iface_attr.bandwidth.shared+dedicated) <= pci_bw
- for iface that is rdma device (with rc/dc/ud/tcp transports), pci_bw is NOT max - which means we really detected it from sysfs

Reply:
The test is running for tcp transports only. We want to move it to some other place?

Reply:
yes, run it for all transports.

Reply:
how can I identify TCP/rdma device?